shardingsphere-sql-parser/shardingsphere-sql-parser-dialect/shardingsphere-sql-parser-mysql/src/main/antlr4/imports/mysql/TCLStatement.g4

```diff
@@ -90,5 +90,17 @@ lockOption
     ;
 
 xid
-    : string_ (COMMA_ string_)* numberLiterals?
+    : gtrid ((COMMA_ bqual) (COMMA_ formatID)?)?
```

Comment:
@zhfeng Maybe it's better like this:
```
xid
    : gtrid=textString (COMMA_ bqual=textString (COMMA_ formatID=NUMBER_)?)?
    ;

```

and delete `gtrid`,`bqual`,`formatID`

Reply:
good catch ! I will check it. Thanks !

Reply:
Done